.github/workflows/release.yaml

```diff
@@ -53,3 +53,26 @@ jobs:
           platforms: linux/amd64,linux/arm64
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Tag and push images for Red Hat certification
```

Comment:
**Question:** How can we make sure this works before releasing?
Did you trick GHA into running this at least once?

Reply:
No. This is just the same code as cass-operator uses (it's copied from there, I just changed the ids).